lib/cli.js

```diff
@@ -305,7 +305,9 @@ module.exports = class Cli {
       .recommendCommands()
       .wrap(yargs.terminalWidth() * 0.70)
       .option('clear', {describe: 'Clears the lando tasks cache', global: true, type: 'boolean'})
-      .help('lando', 'Show help for lando-based options')
+      .help(false)
+      .option('lando', {hidden: true, type: 'boolean'})
```

Comment:
Remove option?

Reply:
@uberhacker kept for backwards compatibility